app/jobs/consume_assessments_job.rb

```diff
@@ -22,7 +22,13 @@ def perform
         end
 
         if message['response_status'].in? %w[opt_out opt_in]
-          handle_opt_in_opt_out(message)
+          begin
+            handle_opt_in_opt_out(message)
+          rescue StandardError => e
+            Rails.logger.info(
+              "ConsumeAssessmentsJob: failure handling opt-in/opt-out message (message response status: #{message['response_status']}), (error: #{e})"
+            )
+          end
```

Comment:
Rescuing `StandardError` is generally considered a bad practice. Do you just want to see what message is causing the Error? Since the original error will be fixed with the safe navigation (as suggested in the other file) it may be better to just log within the `TwilioSender` when the trigger message, from, or to is `nil`.

Reply:
That's fair, I can change that.

Reply:
Updated here: 26670c3
Let me know what you think